internal/pkg/generator/forwarder/generator.go

```diff
@@ -44,15 +45,21 @@ func Generate(clfYaml string, includeDefaultLogStore, includeLegacyForward bool)
 		},
 		FnIncludeLegacyForward: func() bool { return includeLegacyForward },
 		FnIncludeLegacySyslog:  func() bool { return includeLegacySyslog },
+		CLFVerifier: k8shandler.ClusterLogForwarderVerifier{
+			VerifyOutputSecret: func(output *logging.OutputSpec, conds logging.NamedConditions) bool { return true },
+		},
 	}
 	if includeDefaultLogStore {
 		clRequest.Cluster.Spec.LogStore = &logging.LogStoreSpec{
 			Type: logging.LogStoreTypeElasticsearch,
 		}
 	}
+
 	spec, status := clRequest.NormalizeForwarder()
 	log.V(2).Info("Normalization", "spec", spec)
+	_, _ = utils.ToJson(status)
```

Comment:
@igor-karpukhin is there a purpose to this?  Looks like we should remove as there is overhead.  @syedriko maybe we can add this and any other changes into a "clean up" commit

Reply:
I was wondering the same thing, do this check for well-formed JSON?

Reply:
Looks like we need a rebase but please remove.  I don't see any value here unless @igor-karpukhin can articulate otherwise

Reply:
Rebased